x/launchpad/client/cli/tx.go

```diff
@@ -39,22 +40,21 @@ func GetTxCmd() *cobra.Command {
 // CreateSaleCmd broadcast MsgCreateSale.
 func CreateSaleCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create [flags]",
-		Short: "Create or Setup Sale",
+		Use:   `create [flags]`,
+		Short: "Create or Setup a sale",
 		Long: strings.TrimSpace(
-			fmt.Sprintf(`create a new Sale.
-
+			fmt.Sprintf(`Create or Setup a launchpad sale:
 Example:
-$ %s tx launchpad create --sale-file="path/to/sale.json" --from mykey
+$ %s create --sale-file="path/to/sale.json [flags]
 
-Where sale.json contains:
+Sample sale.json file contents:
 {
 	"token-in": "token1",
 	"token-out": "token2",
 	"initial-deposit": "1000token2",
-	"start-time": "2022-05-23T11:17:36.755Z",
-	"duration": 432000s,
-	"treasury": "osmo1r85gjuck87f9hw7l2c30w3zh696xrq0lus0kq6"
+	"start-time": "2022-06-02T11:18:11.000Z",
+	"duration": "432000s",
+	"treasury": "osmo16phc92xevvcvswdew....cr46tn8pneshtsl8"
```

Comment:
why adding the dots?

Reply:
@dayakarreddyn updated it .. may be for not using any full address in example 
but i think it's okay even we use address

should we use valid address ?